client/model-builder/src/test/java/io/smallrye/graphql/client/model/ClientModelBuilderTest.java

```diff
@@ -235,13 +236,40 @@ void namedClientModelTest() throws IOException {
         assertEquals(3, clientModel.getOperationMap().size());
         assertOperation(clientModel,
                 new MethodKey("findAllStringsQuery", new Class[0]),
-                "query findAll { named { findAll } } ");
+                "query NamedFindAll { named { findAll } }");
         assertOperation(clientModel,
                 new MethodKey("findAllStringsName", new Class[0]),
-                "query findAll { named { findAll } } ");
+                "query NamedFindAll { named { findAll } }");
         assertOperation(clientModel,
                 new MethodKey("update", new Class[] { String.class }),
-                "mutation update($s: String) { named { update(s: $s) } } ");
+                "mutation NamedUpdate($s: String) { named { update(s: $s) } }");
+    }
+
+    @Namespace("first/second")
```

Comment:
I haven't been watching the discussion much, so maybe this was already discussed, but wouldn't it be better to declare the namespace components as an array (`@Namespace({"first","second"})`? That would be more typesafe and avoid having to parse it at runtime